libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4131,7 +4117,7 @@ MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration(const mavlink_comm
 
     EXPECT_DELAY_MS(30000);
     if (is_equal(packet.param1,1.0f)) {
-        if (!calibrate_gyros()) {
+        if (!AP::ins().calibrate_gyros()) {
```

Comment:
This won't compile if `!HAL_INS_ENABLED` - we should return `MAV_RESULT_UNSUPPORTED` in that case.